frontend/public/components/modals/_modals.scss

```diff
@@ -50,11 +51,17 @@
 }
 
 .modal-content {
+  background-color: #fff;
```

Comment:
With the pending dark mode changes, we should use PF4 vars for color values.

Reply:
updated